.forge-snapshots/initialize.snap

```diff
@@ -1 +1 @@
-53843
\ No newline at end of file
+74927
```

Comment:
💀

Reply:
This does include the asserts in the test - this isnt the complete initialize cost

Reply:
ah true